### PR TITLE
Update wrangler to v1.0.2

### PR DESCRIPTION
**Pull request by eliyamlevy**
https://github.com/rancher/backup-restore-operator/issues/306

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -35,7 +37,15 @@ func GetEncryptionTransformers(encryptionConfigSecretName string, secrets v1core
 	if !ok {
 		return transformerMap, fmt.Errorf("no encryptionConfig provided")
 	}
-	return encryptionconfig.ParseEncryptionConfiguration(bytes.NewReader(encryptionConfigBytes))
+	err = ioutil.WriteFile("encryptionConfig.yaml", encryptionConfigBytes, os.ModePerm)
```
Comment (jiaqiluo) [inline pkg/util/util.go:40]: I'd suggest adding a `defer` block to delete this file.

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: pkg/util/util.go (hunk 1 above)
Comment (eliyamlevy) [reply pkg/util/util.go:40]: True!

File: pkg/util/util.go [hunk 2]
```diff
@@ -35,7 +37,16 @@ func GetEncryptionTransformers(encryptionConfigSecretName string, secrets v1core
...
```
Comment (jiaqiluo) [inline pkg/util/util.go:40]: Nit: declare the filename as a const at the top of the file instead of a hardcoding value in the code.

**Review by jiaqiluo — APPROVED** [review]
LGTM with one nit.

**Review by jiaqiluo — APPROVED** [review]

**Pull request merged**